workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityGameObjectCommandHandlersGenerator.tt

```diff
@@ -108,13 +110,55 @@ namespace <#= qualifiedNamespace #>
                         return -1;
                     }
 
+                    Action<<#= receivedResponseType #>> wrappedCallback = response =>
```

Comment:
where/how is that used?

Reply:
It's used to not call the thing if the sender that originally sent it is no longer valid

Reply:
but the `wrappedCallback ` is not called anywhere?